packages/neuron-wallet/src/utils/export-history.ts

```diff
@@ -0,0 +1,149 @@
+import fs from 'fs'
+import sqlite3 from 'sqlite3'
+import { get as getDescription } from 'database/leveldb/transaction-description'
+import i18n from 'locales/i18n'
+import shannonToCKB from 'utils/shannonToCKB'
+
+
+interface ExportHistoryParms {
+  walletID: string
+  dbPath: string
+  lockHashList?: string[]
+  filePath: string
+}
+const exportHistory = ({
+  walletID,
+  dbPath,
+  lockHashList = [],
+  filePath
+}: ExportHistoryParms): Promise<number> => {
+  return new Promise((resolve, reject) => {
+    if (!dbPath) {
+      return reject(new Error(`Database is not required`))
```

Comment:
"Database is not required" => "Database is required"?

Reply:
> "Database is not required" => "Database is required"?

Sure, it's a typo.